### DISCUSSION TITLE
Replacement for removed function

### DISCUSSION THREAD

**Discussion by Crizz0** [composer/composer#9614] [Q&A / Support]
Hi,

here is removed the function setOutputProgress() 
https://github.com/composer/composer/pull/7904/files#diff-b99c0f08ed0d8f48cb9926a9f015a5cc1fadc6a5c4111c106d788c64264bcb30L86-L101

Is there any advice how to handle things which used it. Did not find something in the upgrade guide or changelog.

regards

**Comment by rxu** [comment]
Found out it should be `->getInstallationManager()->setOutputProgress()`.
